packages/core/src/textInput/private/Input.tsx

```diff
@@ -11,6 +11,7 @@ type InputProps = {
   iconColor?: string;
   type?: string;
   backgroundColor?: ColorProps;
+  isLabelPresent?: boolean;
```

Comment:
add this to the docs prop table

Reply:
This is an inner prop. Only set by the TextInput internally. Not to be used by outside devs.